lib/elements/clone.py

```diff
@@ -20,18 +20,6 @@
                         XLINK_HREF, SVG_GROUP_TAG)
 from ..utils import cache
 from .element import EmbroideryElement, param
-from .validation import ValidationWarning
-
-
-class CloneWarning(ValidationWarning):
```

Comment:
I think this is still valid. If they want to control cloned elements params wise they should know how to unlink the clone?
Maybe we could rephrase it a little bit though. Since the limits come with the benefit that you only have to update the params of the source element which isn't mentioned at all.

Reply:
Ah, maybe it was a bit too optimistic, huh?~ I'll put it back.